libkirk/main.py

```diff
@@ -501,6 +502,12 @@ def run(cmd_args: list = None) -> None:
         "-x",
         action="store_true",
         help="Force parallelization execution of all tests")
+    parser.add_argument(
+        "--runtime",
+        "-u",
+        type=_time_config,
+        default="0",
+        help="Set for how long we want to run the session (disabled: 0)")
```

Comment:
I would specify it's `sec`

Reply:
Also, why to have disable value? I would expect `-1` would run infinitely.

Reply:
zero or `-1` doesn't really change much. I will remove the disabled comment tho